extensions/adobe/experience/marketo/marketo-person.schema.json

```diff
@@ -0,0 +1,206 @@
+          "type": "string"
+        },
+        "marketo:marketingSuspended": {
+          "title": "Marketing Suspended",
+          "description": "Marketing suspended.",
+          "type": "boolean"
+        },
+        "marketo:marketingSuspendedCause": {
+          "title": "Marketing Suspended Cause",
+          "description": "Marketing suspended cause.",
+          "type": "string"
+        },
+        "marketo:notes": {
+          "title": "Notes",
+          "description": "Notes.",
+          "type": "string"
+        },
+        "marketo:orgID": {
+          "title": "Organization ID",
+          "description": "Organization id.",
+          "type": "string"
+        },
+        "marketo:primaryLeadInterestID": {
+          "title": "Primary Lead Interest Id",
+          "description": "Primary lead interest id.",
+          "type": "boolean"
+        },
+        "marketo:registerSourceInfo": {
+          "title": "Register Source Info",
+          "description": "Register source info.",
+          "type": "string"
+        },
+        "marketo:registerSourceType": {
+          "title": "Register Source Type",
+          "description": "Register source type.",
+          "type": "string"
+        },
+        "marketo:salesContactOwnerID": {
+          "title": "Sales Contact Owner Id",
+          "description": "Sales contact owner id.",
+          "type": "string"
+        },
+        "marketo:originalReferrer": {
+          "title": "Original Referrer",
+          "description": "The original referrer is typically the page that leads to a form fill out.",
+          "type": "string"
+        },
+        "marketo:originalSearchEngine": {
+          "title": "Original Search Engine",
+          "description": "Search engine that the original referrer used.",
+          "type": "string"
+        },
+        "marketo:originalSearchPhrase": {
+          "title": "Original Search Phrase",
+          "description": "Search phrase used by the original referrer.",
+          "type": "string"
+        },
+        "marketo:originalSourceInfo": {
+          "title": "Original Source Info",
+          "description": "Original source info.",
+          "type": "string"
+        },
+        "marketo:originalSourceType": {
+          "title": "Original Source Type",
+          "description": "Original ource type.",
+          "type": "string"
+        },
+        "marketo:leadAddress": {
+          "title": "Lead Address",
+          "description": "Lead address.",
+          "$ref": "https://ns.adobe.com/xdm/common/address"
+        },
+        "marketo:leadScore": {
+          "title": "Lead Score",
+          "description": "Lead score.",
+          "type": "number"
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "#/definitions/marketo-person"
+    }
+  ],
+  "meta:status": "experimental"
+}
```

Comment:
Missing required marketo:ContactID. Needs a separation of marketo-lead, for lead specific attributes as "ContactID" will not be part of lead and we need it here as required property.